forum/qa-content/qa-page.js

```diff
@@ -177,6 +177,58 @@ function qa_ajax_error()
 }
 
 
+/**
+ *	Feature: check if browser supports copying to clipboard.
+ *	If true, return function, which can copy highlighted content to clipboard on button click - then user can paste it wherever he wants into.
+ *	Otherwise return null.
+ */
+window.initializeCopyToClipboard = ( function() {
+	// check if browser supports 'select()' and 'copy' commands
+	var isClipboardSupported = ( window.getSelection && document.queryCommandSupported( 'copy' ) && navigator.userAgent.indexOf( 'Firefox' ) === -1 );
+
+	function copyToClipBoard( ev, contentToCopy ) {
+		// prevent page refresh (or something weird) as default button action
+		ev.preventDefault();
+
+		if ( !ev.defaultPrevented ) {
+			return false;
+		}
+
+		/*
+		 * In order to be able to copy the code inside block into the clipboard - so user can easily paste it wherever he wants - within single button click
+		 * a code must be first selected (or highlighted in human meaning), so JavaScript can copy it.

```

Comment:
In two first lines you basically paraphrase the comment  above `initializeCopyToClipboard`. This whole essay can be shortened to:

>XXX: Selection only works on inputs/textarea, thus, we create hidden <textarea>, insert there relevant contents, copy it to clipboard and remove the element afterwards.

Comments should be concise and avoided when the code is self-explanatory.
XXX is a common pratice to indicate that some hack is happening in the code below.